### PR TITLE
Added new test scenarios for websocket

### PR DESCRIPTION
**Pull request by jhkimnew**
@pan-wang Please approve this for the new test scenarios for websocket scenarios and fixing build issues.

### CODE REVIEW

File: test/AspNetCoreModule.Test/Framework/IISConfigUtility.cs [hunk 1]
```diff
@@ -254,7 +271,7 @@ public void RecycleAppPool(string appPoolName)
             using (ServerManager serverManager = GetServerManager())
             {
                 serverManager.ApplicationPools[appPoolName].Recycle();
-            }
+            }
```
Comment (pan-wang) [inline test/AspNetCoreModule.Test/Framework/IISConfigUtility.cs]: Remove the whitespaces

File: test/AspNetCoreModule.Test/Framework/TestUtility.cs [hunk 2]
```diff
@@ -811,7 +851,7 @@ public static int RunCommand(string fileName, string arguments = null, bool chec
             p.StartInfo.UseShellExecute = false;
             p.StartInfo.CreateNoWindow = true;            
             p.Start(); 
-            pid = p.Id;
+            pid = p.Id;
```
Comment (pan-wang) [inline test/AspNetCoreModule.Test/Framework/TestUtility.cs]: Remove the whitespaces

File: test/AspNetCoreModule.Test/Framework/TestWebSite.cs [hunk 3]
```diff
@@ -309,41 +353,126 @@ public void StartIISExpress(string verificationCommand = null)
...
                 {
-                    isIISExpressReady = true;
-                    break;
+                    throw new System.ApplicationException("Not found :" + debuggerCmdline + "; this test requires windbg.exe.");
                 }
-                else
+            }
+            else
+            {
+                debuggerCmdline = Path.Combine(Environment.ExpandEnvironmentVariables("%ProgramFiles%"), "Debugging Tools for Windows (x64)", "windbg.exe");
```
Comment (pan-wang) [inline test/AspNetCoreModule.Test/Framework/TestWebSite.cs:414]: you hardcode the path of WinDBG with default installation path. As user may change the installation path, you may want to  run query such as https://stackoverflow.com/questions/908850/get-installed-applications-in-a-system

**Review by pan-wang — APPROVED** [review]

File: test/AspNetCoreModule.Test/Framework/TestWebSite.cs (hunk 3 above)
Comment (jhkimnew) [reply test/AspNetCoreModule.Test/Framework/TestWebSite.cs:414]: I have a plan to update whole tests can be started with attaching debugger and appverifer rule. At that time, I will remove the hardcode path as you suggest.

**Pull request merged**